yt_dlp/extractor/curiositystream.py

```diff
@@ -23,9 +23,11 @@ def _call_api(self, path, video_id, query=None):
             auth_cookie = self._get_cookies('https://curiositystream.com').get('auth_token')
             if auth_cookie:
                 self.write_debug('Obtained auth_token cookie')
-                self._auth_token = auth_cookie.value
+                self._auth_token = compat_urllib_parse_unquote(auth_cookie.value)
+            else:
+                self.write_debug('Auth cookie NOT found')
         if self._auth_token:
-            headers['X-Auth-Token'] = self._auth_token
+            headers['x-auth-token'] = self._auth_token
```

Comment:
why the change?

Reply:
That's what official front-end is sending. If I remember correctly, headers' names *should* be case-insenstitive, but back-ends are not always adhering to it.

Reply:
Our core code/urllib will not respect the case you set here, making this change unnecessary
```suggestion
            headers['X-Auth-Token'] = self._auth_token
```